package.json

```diff
@@ -34,6 +34,7 @@
     "github-markdown-css": "^3.0.1",
     "html-webpack-plugin": "^3.2.0",
     "is-empty-object": "^1.1.1",
+    "jsonpipe": "2.2.0",
```

Comment:
Currently, I have found 2 libraries for HTTP requests. The second one is `whatwg-fetch`. Let's use only one

Reply:
`jsonpipe` is not exactly a library for HTTP requests. This library is aimed specifically at handling 'chunked transfer encoding': it receives HTTP chunks (note that those chunks are not necessarily full json objects), concatenates them, splits the resulting array by delimiter (it does it on the fly without waiting for all data) and then invokes callback for each parsed json object. This library should not be used for regular http requests (although there is a possibility that you technically can use it for regular requests, it is definitely not meant for that). [github](https://github.com/eBay/jsonpipe)
On the other hand, `whatwg-fetch` does not support streaming at all.
Therefore, we have to use these two libraries at the same time.